docs/reference/docs/reindex.asciidoc

```diff
@@ -452,10 +450,36 @@ POST _reindex
 // CONSOLE
 // TEST[setup:host]
 // TEST[s/^/PUT source\n/]
-// TEST[s/otherhost:9200",/\${host}"/]
-// TEST[s/"username": "user",//]
-// TEST[s/"password": "pass"//]
+// TEST[s/otherhost:9200/\${host}/]
 
+It is also possible to set the read timeout on the remote connection with the
+`timeout` field (defaults to one minute):
+
+[source,js]
+--------------------------------------------------
+POST _reindex
+{
+  "source": {
+    "remote": {
+      "host": "http://otherhost:9200",
+      "timeout": "10m"
```

Comment:
can we not have 10 minutes in the example? Something more reasonable, like 30 seconds or one minute?